src/main/java/uk/nhs/tis/trainee/notifications/dto/UserDetails.java

```diff
@@ -40,6 +42,18 @@ public record UserDetails(
     String title,
     String familyName,
     String givenName,
-    String gmcNumber) {
+    String gmcNumber,
+    List<String> role) {
+
+  public UserDetails(
+      Boolean isRegistered,
+      String email,
+      String title,
+      String familyName,
+      String givenName,
+      String gmcNumber
+  ) {
+    this(isRegistered, email, title, familyName, givenName, gmcNumber, List.of());
```

Comment:
Included to allow existing instantiation without the role to be left as-is, but this could be an unexpected 'feature'.